srcopsmetrics/entities/pull_request.py

```diff
@@ -94,11 +98,11 @@ def store(self, pull_request: GithubPullRequest):
             "closed_by": closed_by,
             "merged_at": merged_at,
             "commits_number": commits,
-            "referenced_issues": PullRequest.get_referenced_issues(pull_request),
-            "interactions": GitHubKnowledge.get_interactions(pull_request.get_issue_comments()),
-            "reviews": self.extract_pull_request_reviews(pull_request),
-            "requested_reviewers": self.extract_pull_request_review_requests(pull_request),
-            "labels": GitHubKnowledge.get_labels(pull_request.as_issue()),
+            # "referenced_issues": PullRequest.get_referenced_issues(pull_request),
```

Comment:
why these are commented?

Reply:
nice catch! these features are being changed in #377 , so I can revert changes back here.